### ISSUE TITLE
Submit into official repository

### ISSUE THREAD

**Issue by YT-Advanced** [valinet/wh-mods#2]
I think it's best to have all mods in the official repository, it's more convenient for users to install, update, and see insights such as the amount of users.

**Comment by valinet** [comment]
Thanks for the suggestion. Well, firstly, I personally am not a fan of everyone committing everything into a single folder of a particular repo. Long term, I envision Windhawk receiving some kind of "sources" support, where users can add entries to a list of URLs the app pulls mods from - those URLs would probably be git repos formatted a specific way, maybe? Think of it like sources in Cydia (iOS) or AltStore (iOS) etc. That being said, I am not pushy for this either - I understand that Michael might have some other stuff to work on at the moment, and as a matter of fact, I do too, as otherwise I would have at least tried contributing the infrastructure around that to the project already.

Lastly, mods here are more like quick'n'dirty experiments that I use on some of my machines, idk if they necesarily belong to the main repo, also motivated by what I explained above. There are some philosophical differences still with the main repo: for example, @aubymori has a gigantic awesome mod that replicates OldNewExplorer's functionality, but it is an all in one package. So, for my use case, I just took out the parts that interested me and made separate mods, as that is how I like to roll on my machines. Yeah, idk, to each their own, what is important is that we have more free and open resources than ever on this patform for people to play around with, and a great framework that we can use to toy around with this stuff (and not only), so yeah... users can come here, take whatever they want from here and do whatever they want with it. And tbh, I don't have much time to maintain it either, which moving to the main repo would kind of necessitate paying more attention into.

Tl;dr These living here I believe is fine for now.
